index.js

```diff
@@ -24,6 +30,24 @@ app.use('/api', userRouter);
 app.use('/api', fieldOfActivityRouter);
 app.use('/api', companyRouter);
 app.use('/api', signinRouter);
+app.use('/api', (req, res, next) => {
```

Comment:
Please create this middleware in a separate file, in `middlewares` folder and just use it here.